go/backup/backup.go

```diff
@@ -193,6 +193,7 @@ func retrieveStreams(url string) map[string]stream {
 	}
 	if res.StatusCode != 200 {
 		logError("unexpected response: %d", res.Status)
+		ioutil.ReadAll(res.Body)
```

Comment:
This still has the `defer res.Body.Close()` after the first return.

Reply:
Changed the pattern throughout the code base